conduct/code_of_conduct.md

```diff
@@ -0,0 +1,123 @@
+
+# Project Jupyter Code of Conduct and Community Guidelines
+
+Project Jupyter is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission and to create an open platform that serves a broad range of
+communities, from research and education, to journalism, industry and
+beyond. This diversity is one of our biggest strengths, but it can also
+lead to communication issues and conflicts. Therefore, we have a few
+ground rules we ask that our community members adhere to.
+
+Fundamentally, we are committed to providing a productive,
+harassment-free environment for everyone. Rather than considering this
+code an exhaustive list of things that you can’t do, take it in the
+spirit it is intended - a guide to make it easier to enrich all of us
+and the communities in which we participate.
+
+This code applies equally to founders, developers, mentors and new
+community members, in all spaces managed by Project Jupyter. This
+includes the mailing lists, our GitHub organizations, our chat rooms,
+in-person events, and any other forums created by the project team. In
+addition, violations of this code outside these spaces may affect a
+person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you
+report it by emailing
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) or by filling out
+[this form](https://goo.gl/forms/sJzOIie3zde9M71T2). For more details
+please see our Reporting Guidelines.
+
+By embracing the following principles, guidelines and actions to follow
+or avoid, you will help us make Jupyter a welcoming and productive
+community.
+If that doesn't answer your questions, feel free to contact us at
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org).
+
+
```

Comment:
Note: We should list _who_ will receive the mail if sent at `conduct@...` and have a way to report if a person that the complaint is against is on this list.